pkg/commands/workload_test.go

```diff
@@ -1675,6 +1675,7 @@ Published source
 			cmd := &cobra.Command{}
 			ctx := cli.WithCommand(context.Background(), cmd)
 			ctx = source.StashGgcrRemoteOptions(ctx, remote.WithTransport(registry.Client().Transport))
+			ctx = source.StashClientTransport(ctx, registry.Client().Transport)
```

Comment:
No need to do both lines 1677 and 1678, pick one.

Reply:
Changed stash methods